fn do_something() {
    expected_error();
}